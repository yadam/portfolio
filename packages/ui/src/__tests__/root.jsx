import React from 'react';
import renderer from 'react-test-renderer';
import Root from '../root';

it('renders correctly', () => {
  const tree = renderer.create(<Root />).toJSON();
  expect(tree).toMatchSnapshot();
});
