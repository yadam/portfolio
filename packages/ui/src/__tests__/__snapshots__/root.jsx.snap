// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10"
>
  <div
    className="MuiGrid-container-37 MuiGrid-spacing-xs-24-61"
  >
    <div
      className="MuiGrid-item-38 MuiGrid-grid-xs-6-71"
    >
      <div
        className="MuiPaper-root-134 MuiPaper-elevation2-138 MuiPaper-rounded-135"
      >
        Hello World!
      </div>
    </div>
  </div>
</p>
`;
